File: Python/percobaan4.py
options = {
    1: "Senin",
    2: "Selasa",
    3: "Rabu",
    4: "Kamis",
    5: "Jumat",
    6: "Sabtu",
    7: "Minggu",
} 
x = float(input("Masukan angka yang Anda(1 - 7): "))
print("Hari: %s" %options.setdefault(x, "Inputan hanya 1-7"))  

File: C++/percobaan4.cpp
#include <iostream>
using namespace std;

int main() {
  int angka;
  cout << "Masukan Case(1-3) yang Anda Inginkan: ";
  cin >> angka;
  switch (angka)
  {
  case 1:
      cout << "Ini Case 1 dan Anda mendapat uang $1,000,000" << endl;
      break;
  case 2:
      cout << "Ini Case 2 dan Anda mendapat motor ferrari" << endl;
      break;
  default:
      cout << "Ini Case 2 dan Anda mendapat motor ferrari" << endl;
      break;
  }
  return 0;
}
